src/zenml/client.py

```diff
@@ -364,6 +364,30 @@ def _reset_instance(cls, client: Optional["Client"] = None) -> None:
         """
         cls._global_client = client
 
+    @staticmethod
+    def _fail_for_sql_zen_store(method):
+        """Decorator for all methods, that are disallowed when the client is not connected through REST API.
+
+        Args:
+            method: The method
+
+        Returns:
+            The decorated method.
+        """
+
+        @functools.wraps(method)
+        def wrapper(self, *args, **kwargs):
+            # No isinstance check to avoid importing ZenStore implementations
+            breakpoint()
```

Comment:
I think this shouldn't be here :)

Reply:
This is definitely a necessary part :stuck_out_tongue: